multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/util/FileUtils.java

```diff
@@ -72,6 +72,16 @@ public static void cleanUp(Path filePath, Logger logger) {
         }
     }
 
+    public static boolean isFilenameValid(String file) {
+        File f = new File(file);
+        try {
+            f.getCanonicalPath();
```

Comment:
When will this throw an exception? Also, isn't there anything here that would do the job:
https://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/FilenameUtils.html

Reply:
Changed to `FilenameUtils::validatePath`